content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+---
+title: "Blue Boot2Root Write-Up"
+date: 2020-11-11T13:43:28-04:00
+author: "Nate Catelli"
+tags: ["ctf", "boot2root", "hacking", "writeup", "tryhackme"]
+description: "A boot2root writeup of the Blue host from TryHackMe"
+type: "post"
+draft: false
+---
+
+## Introduction:
+After participating in the Disney Can-You-Hack-It CTF, I've been trying to spend more of my time studying security and offensive penetration testing. Primarily, I've been trying to balance both understanding how to effectively perform an attack with the awesome frameworks out their like metasploit, but also gain a better understanding of what this framework is doing under the hood. This post is meant to be the first of many as I attempt to practice documenting both the attack and my methodology.
```

Comment:
> Primarily, I've been trying to balance both understanding how to effectively perform an attack with the awesome frameworks out their like metasploit, but also gain a better understanding of what this framework is doing under the hood.

My main goal is to balance understanding how to effectively perform an attack with the awesome frameworks out there like metasploit with gaining a better understanding of what this framework is doing under the hood.